src/components/Icons/Icon.stories.tsx

```
@@ -0,0 +1,135 @@
+export default meta;
+
+type Story = StoryObj<
+  | typeof IconCheck
+  | typeof IconChevron
+  | typeof IconConfirmCircle
+  | typeof IconEdit
+  | typeof IconErrorCircle
+  | typeof IconPictureSquare
+>;
+
+const IconShowcase = ({IconComponent, args}: {IconComponent: any; args: any}) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)